google-cloud-debugger/Rakefile

```diff
@@ -13,12 +13,14 @@ Rake::ExtensionTask.new(ext_name, spec)
 namespace :compile do
   desc "Cleanup compilation files"
   task :cleanup do
-    puts "remove #{ext_name}.*"
     Dir.glob "lib/google/cloud/debugger/debugger_c.*" do |f|
+      puts "remove #{f}"
       File.delete f
     end
-    puts "remove directory tmp"
-    FileUtils.remove_dir "tmp"
+    if File.directory? "tmp"
+      puts "remove directory ./tmp"
+      FileUtils.remove_dir "tmp"
```

Comment:
Instead of checking explicitly for existence, just pass true for the "force" parameter (i.e. `FileUtils.remove_dir "tmp", true`).